src/content/docs/release-notes/agent-release-notes/ruby-release-notes/ruby-agent-870.mdx

```diff
@@ -3,7 +3,7 @@ subject: Ruby agent
 releaseDate: '2022-05-03'
 version: 8.7.0
 downloadLink: 'https://rubygems.org/downloads/newrelic_rpm-8.7.0.gem'
-features: ["APM logs-in-context log forwarding on by default", "Improved async support and Thread instrumentation"]
+features: ["APM logs-in-context log forwarding on by default", "Improved async support and Thread instrumentation", "Deprecate support for Ruby 2.2", "Deprecate support the following technology versions lower than: ActiveMerchant 1.64.0, ActiveRecord 3.2.22.5, Bunny 2.6.7, Dalli 2.7.6, Delayed Job 4.1.2, Excon 0.55.0, Grape 0.19.1, HTTPClient 2.8.2.4, HTTP.rb 2.2.1, Mongo 2.4.0, Padrino 0.14.4, Passenger 5.1.2, Puma 3.8.2, Rack 1.6.7, Rails 3.2.22.5, Sequel 4.44.0, Sidekiq 4.2.10, Sinatra 1.4.8, Typhoeus 1.1.2, Unicorn 5.2.0", "Deprecate support for all versions of Acts As Solr, Authlogic, DataMapper, Rainbows, and Sunspot"]
```

Comment:
The line with all the technologies seems really long, but I think this will be a good test of the UI's abilities and perhaps help find a required length going forward.